api-description/forms/pk/post.json

```diff
@@ -2,22 +2,36 @@
     "description": "Endpoint to answer a form",
     "url": "http://localhost:8000/forms/1",
     "fields": {
-        "answers": {
-            "1": "Friends",
-            "2": "one_time",
-            "3": ["two_times", "three_times"],
-            "4": 5
-        }
+        "survey": 1,
+        "user_responses": [
+            {
+                "fieldset": 1,
+                "user_id": "User#20#",
```

Comment:
So we're not basing this on `csrf` tokens? Also It needs to be resend per `fieldset`?